lab-padget/test/hello-test.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+console.log('evaluating hello.js');
+
+const assert = require('assert');
+const hi = require('../lib/hello.js');
+let name = 'Michael';
```

Comment:
What was your reasoning for using let vs const here? Not saying you're wrong, but I'm curious about your logic.